drake/examples/Cars/simulateLCM.cpp

```diff
@@ -78,11 +89,22 @@ int main(int argc, char* argv[]) {
   // be reused
   DrakeJoint::FloatingBaseType floating_base_type = DrakeJoint::QUATERNION;
 
-
   auto rigid_body_sys = make_shared<RigidBodySystem>();
-  rigid_body_sys->addRobotFromFile(argv[1], floating_base_type);
-  auto const & tree = rigid_body_sys->getRigidBodyTree();
-  for (int i=2; i<argc; i++)
+
+  // The following variable, weld_to_frame, is only needed if the model is a
+  // URDF file. It is needed since URDF does not specify the location and
+  // orientation of the car's root node in the world. If the model is an SDF,
+  // weld_to_frame is ignored by the parser.
```

Comment:
i do want to explore the case where we can separate the dynamics, but in general you cannot.  if multiple robots can interact dynamically, then right now we need to allow them to live in the same rigidbodytree.

going forward, i do think we should explore the generalize force inputs in my ideal class design (which is documented here: https://github.com/RobotLocomotion/drake/blob/master/drake/systems/plants/RigidBodySystem.h#L10) as a means of coupling robots that only interact (with contact forces, or by sharing state -- eg. the multiple cars interacting with each other) in a sparse way.

i wouldn't try to touch those issues in this PR